loginpage/index.html

```diff
@@ -48,14 +79,37 @@
     <br><br><br>
     <br>
     <p style="margin-left:30%;">Log in to Loklak!</p>
+=======
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+   <div class="container-fluid">
+    <div class="navbar-header">
+     <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
```

Comment:
Line is longer than allowed. (149 > 129)

*LineLengthBear, severity NORMAL, section `default`.*